Problem2.py

```diff
@@ -0,0 +1,40 @@
+'''
+Min Heap
+'''
+
+from heapq import heappush, heappop, heapify
+
+class Solution:
+    def __init__(self):
+        self.myHeap = []
+        heapify(self.myHeap)
+        
+    def insert(self, item):
+        # Time complexity: O(logn)
+        heappush(self.myHeap, item)
+        
+    def getMin(self):
+        # Time complexity: O(1)
+        return self.myHeap[0]
+        
+    def extractMin(self):
+        # Time complexity: O(logn)
+        return heappop(self.myHeap)
+        
+    def printHeap(self):
+        print(self.myHeap)
```

Comment:
You are not supposed to use the `heapq` module to implement MinHeap. Please implement MinHeap using Arrays only.